packages/react-components/react-components/src/AccessibilityScenarios/Input.stories.tsx

```diff
@@ -184,258 +174,246 @@ const TicketOrderFormInputsAccessibility = () => {
+            control={control}
+            as={
+              <Input
+                type="text"
+                id="birthDate"
+                placeholder="E.g. 3/21/1995"
+                aria-required="true"
+                aria-invalid={!!errors.birthDate}
+                aria-describedby="birthDateErrors"
+              />
+            }
+            rules={{
+              required: true,
+              validate: {
+                validDate: value => regexes.validDate.test(value),
+                always: () => {
+                  if (!formState.isSubmitting) {
+                    formValidation.onFieldValidated('birthDate');
+                  }
+                  return true;
+                },
+              },
+            }}
+          />
+          {errors.birthDate?.types && (
+            <ValidationMessage id="birthDate" formValidation={formValidation}>
+              {'required' in errors.birthDate.types ? (
+                <p>Birth date is required.</p>
+              ) : (
+                <>
+                  <p>Birth date is invalid. It must:</p>
+                  <ul>{'validDate' in errors.birthDate.types && <li>Be in the MM/DD/YYYY format.</li>}</ul>
+                </>
+              )}
+            </ValidationMessage>
+          )}
+
+          <Checkbox label="Send me newsletter" onChange={onSendNewsletterChange} />
+
+          <Label htmlFor="email">E-mail:</Label>
+          <Controller
+            name="email"
+            control={control}
+            as={
+              <Input
+                type="text"
+                id="email"
+                disabled={!isSendNewsletter}
+                aria-required={isSendNewsletter}
+                aria-invalid={!!errors.email}
+                aria-describedby="emailErrors"
+              />
+            }
+            rules={{
+              required: isSendNewsletter,
+              validate: {
+                validEmail: value => regexes.validEmail.test(value),
+                always: () => {
+                  if (!formState.isSubmitting) {
+                    formValidation.onFieldValidated('email');
+                  }
+                  return true;
+                },
+              },
+            }}
+          />
+          {errors.email?.types && (
+            <ValidationMessage id="email" formValidation={formValidation}>
+              {'required' in errors.email.types ? (
+                <p>E-mail is required.</p>
+              ) : (
+                <>
+                  <p>E-mail is invalid. It must:</p>
+                  <ul>
+                    {'validEmail' in errors.email.types && <li>Be a valid e-mail address, like name@example.com.</li>}
+                  </ul>
+                </>
+              )}
+            </ValidationMessage>
+          )}
+
+          <Label htmlFor="securityCode">Your security code:</Label>
+          <Input type="text" id="securityCode" value={generateSecurityCode()} readOnly />
```

Comment:
The result of `generateSecurityCode` here should also be memoized so it doesn't change